phpBB/language/en/acp/board.php

```diff
@@ -50,6 +50,11 @@
 	'DEFAULT_STYLE_EXPLAIN'			=> 'The default style for new users.',
 	'DISABLE_BOARD'					=> 'Disable board',
 	'DISABLE_BOARD_EXPLAIN'			=> 'This will make the board unavailable to users who are neither administrators nor moderators. You can also enter a short (255 character) message to display if you wish.',
+	'DISABLE_BOARD_ACCESS'			=> 'Limit access to disabled board',
+	'DISABLE_BOARD_ACCESS_EXPLAIN'	=> 'This setting limits the access to a disabled board to only administrators, or administrators plus global moderators or administrators and all moderators.',
```

Comment:
I think the menu options speak for themselves. Maybe just say: "This setting limits who can access a disabled board."

Reply:
Yes, good point. Will change this, but i wait for potential more feedback first.